pkgs/development/python-modules/sharp_aquos_rc/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "sharp_aquos_rc";
```

The name should be normalized according to [PEP-503](https://nixos.org/manual/nixpkgs/stable/#contributing).

```suggestion
  pname = "sharp-aquos-rc";
```

---

The same goes for the attribute and directory name.

---

Does the Home Assistant update script need updated as well? It generates `component-packages.nix` with `sharp_aquos_rc`.

---

It ignores these inaccuracies. But yes, it needs to be recreated.